### PR TITLE
Estimation dec 2020

### PR DESCRIPTION
**Pull request by boryana-ilieva**
Added moments to identify better disutility parameter following blundell et. al Supplement pp. 14-16

### CODE REVIEW

**Comment by boryana-ilieva** [comment]
would be good to know if there is some sort of mistake in the calculation of transitions from out of work into work. While Blundell et. al. (supplement p. 15 have numbers between 18 and 50%, we have numbers strictly below 6.5%. To be fair their numbers seems less reasonable to me than ours in this case...

File: python/SimulationBasedEstimation.py [hunk 1]
```diff
@@ -89,18 +89,20 @@ def _calculate_criterion_func_value(self, params_cand):
...
                 stats_obs.extend(self.moments_obs[group][key_])
                 stats_sim.extend(moments_sim[group][key_])
 
+        stats_obs = np.ndarray.flatten(np.array(stats_obs))
+        stats_sim = np.ndarray.flatten(np.array(stats_sim))
+
         # Construct criterion value
-        stats_dif = np.array(stats_obs) - np.array(stats_sim)
+        stats_dif = stats_obs - stats_sim
+        print(stats_dif)
```
Comment (peisenha) [inline python/SimulationBasedEstimation.py]: Pleaser remove print statements everywhere.

File: estimation/basecamp/moments_calculation.py [hunk 2]
```diff
@@ -46,3 +48,455 @@ def get_moments(data):
             moments["Choice_Probability"][period].append(stat)
 
     return moments
+
+
+def transitions_out_to_in(data_subset):
+    counts_list = []
+    for period in np.arange(1, 39):
```
Comment (peisenha) [inline estimation/basecamp/moments_calculation.py]: ` 39` is hard-coded over and over, please make endogenous.

**Review by peisenha — APPROVED** [review]

File: python/SimulationBasedEstimation.py (hunk 1 above)
Comment (boryana-ilieva) [reply python/SimulationBasedEstimation.py]: Done

File: estimation/basecamp/moments_calculation.py (hunk 2 above)
Comment (boryana-ilieva) [reply estimation/basecamp/moments_calculation.py]: Done

**Pull request merged**